### ISSUE TITLE
No predictions when training or testing the net !!!

### ISSUE THREAD

**Issue by hualitlc** [Holmeyoung/crnn-pytorch#64]
**Following is the middle results of the training process.**

> Val loss: 0.007887, accuray: 0.000049
[17/1000][10/109] Loss: 0.006624
[17/1000][20/109] Loss: 0.006612
[17/1000][30/109] Loss: 0.006566
[17/1000][40/109] Loss: 0.006557
[17/1000][50/109] Loss: 0.006572
[17/1000][60/109] Loss: 0.006588
[17/1000][70/109] Loss: 0.006571
[17/1000][80/109] Loss: 0.006521
[17/1000][90/109] Loss: 0.006565
[17/1000][100/109] Loss: 0.006536
Start val
-------------------------- =>                     , gt: 蓝芯体验
-------------------------- =>                     , gt: 实物图
-------------------------- =>                     , gt: 13240796631
-------------------------- =>                     , gt: 装
-------------------------- =>                     , gt: 附教学光盘
-------------------------- =>                     , gt: 排行榜
-------------------------- =>                     , gt: VOLVO
-------------------------- =>                     , gt: 防漏
-------------------------- =>                     , gt: 全车配件销售
-------------------------- =>                     , gt: 3
Val loss: 0.008278, accuray: 0.000049
[18/1000][10/109] Loss: 0.006544
[18/1000][20/109] Loss: 0.006473
[18/1000][30/109] Loss: 0.006494
[18/1000][40/109] Loss: 0.006514
[18/1000][50/109] Loss: 0.006504
[18/1000][60/109] Loss: 0.006405
[18/1000][70/109] Loss: 0.006445
[18/1000][80/109] Loss: 0.006362
[18/1000][90/109] Loss: 0.006427
[18/1000][100/109] Loss: 0.006379
Start val
-------------------------- =>                     , gt: 实物照片
-------------------------- =>                     , gt: BeJiRog
-------------------------- =>                     , gt: 34N
-------------------------- =>                     , gt: 售后无忧
-------------------------- =>                     , gt: 诚信品质实力
-------------------------- =>                     , gt: 陪伴宝宝益智成长
-------------------------- =>                     , gt: PBA:G25238-300
-------------------------- =>                     , gt: 全国包邮
-------------------------- =>                     , gt: Mrcake蛋糕坊
-------------------------- =>                     , gt: 磨粉

**And I print the preds, which is as following:
 preds = crnn(image)
 print(preds)**

`Start val
tensor([[[-0.0718, -4.1044, -4.4439,  ..., -4.0910, -4.5487, -4.5314],
         [-0.0719, -4.1049, -4.4443,  ..., -4.0916, -4.5493, -4.5318],
         [-0.0719, -4.1053, -4.4446,  ..., -4.0917, -4.5495, -4.5323],
         ...,
         [-0.0719, -4.1050, -4.4444,  ..., -4.0917, -4.5494, -4.5320],
         [-0.0718, -4.1035, -4.4431,  ..., -4.0903, -4.5479, -4.5306],
         [-0.0719, -4.1052, -4.4446,  ..., -4.0917, -4.5495, -4.5322]],

        [[-0.1057, -4.8053, -5.0113,  ..., -4.6961, -5.1657, -5.1004],
         [-0.1057, -4.8056, -5.0116,  ..., -4.6964, -5.1660, -5.1007],
         [-0.1058, -4.8060, -5.0120,  ..., -4.6967, -5.1663, -5.1011],
         ...,
         [-0.1057, -4.8060, -5.0119,  ..., -4.6967, -5.1664, -5.1011],
         [-0.1056, -4.8048, -5.0108,  ..., -4.6957, -5.1652, -5.0999],
         [-0.1058, -4.8064, -5.0123,  ..., -4.6971, -5.1667, -5.1015]],

        [[-0.1121, -4.9228, -5.1042,  ..., -4.7994, -5.2690, -5.1971],
         [-0.1121, -4.9231, -5.1044,  ..., -4.7997, -5.2693, -5.1973],
         [-0.1123, -4.9240, -5.1054,  ..., -4.8005, -5.2703, -5.1983],
         ...,
         [-0.1122, -4.9243, -5.1057,  ..., -4.8009, -5.2706, -5.1987],
         [-0.1119, -4.9215, -5.1028,  ..., -4.7983, -5.2677, -5.1956],
         [-0.1123, -4.9243, -5.1057,  ..., -4.8007, -5.2705, -5.1986]],

        ...,

        [[-0.1127, -4.9321, -5.1003,  ..., -4.8029, -5.2650, -5.1939],
         [-0.1131, -4.9340, -5.1024,  ..., -4.8049, -5.2672, -5.1963],
         [-0.1131, -4.9348, -5.1033,  ..., -4.8056, -5.2682, -5.1972],
         ...,
         [-0.1134, -4.9360, -5.1047,  ..., -4.8068, -5.2697, -5.1988],
         [-0.1126, -4.9311, -5.0992,  ..., -4.8020, -5.2638, -5.1927],
         [-0.1133, -4.9362, -5.1050,  ..., -4.8070, -5.2700, -5.1991]],

        [[-0.1122, -4.8674, -5.0035,  ..., -4.7231, -5.1638, -5.0904],
         [-0.1123, -4.8667, -5.0026,  ..., -4.7224, -5.1628, -5.0894],
         [-0.1124, -4.8687, -5.0052,  ..., -4.7246, -5.1656, -5.0923],
         ...,
         [-0.1125, -4.8685, -5.0049,  ..., -4.7243, -5.1654, -5.0920],
         [-0.1122, -4.8666, -5.0026,  ..., -4.7223, -5.1628, -5.0894],
         [-0.1125, -4.8692, -5.0059,  ..., -4.7252, -5.1664, -5.0931]],

        [[-0.1118, -4.4668, -4.4260,  ..., -4.2436, -4.5748, -4.4685],
         [-0.1118, -4.4662, -4.4251,  ..., -4.2429, -4.5739, -4.4674],
         [-0.1120, -4.4680, -4.4277,  ..., -4.2451, -4.5766, -4.4704],
         ...,
         [-0.1121, -4.4680, -4.4276,  ..., -4.2451, -4.5767, -4.4704],
         [-0.1118, -4.4661, -4.4250,  ..., -4.2427, -4.5736, -4.4672],
         [-0.1120, -4.4682, -4.4279,  ..., -4.2453, -4.5769, -4.4707]]],
       device='cuda:0')‘

I want to know the reason, thank you!
